pkgs/development/tools/misc/coreboot-toolchain/update.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env nix-shell
+#!nix-shell --pure -i bash update-shell.nix
+
+set -xe
+
+cd "$(dirname "$0")/../.."
+
+nix-update coreboot $@
+
+src="$(nix-build --no-out-link -A coreboot.src)"
+urls=$($src/util/crossgcc/buildgcc -u)
+
+echo '{ fetchurl }: [' > pkgs/coreboot/.files.nix.tmp
+
+for url in $urls
+do
+	name="$(basename $url)"
+	hash="$(nix-prefetch-url "$url")"
```

Comment:
```suggestion
for url in $urls; do
  name="$(basename $url)"
  hash="$(nix-prefetch-url "$url")"
```
Please use spaces instead of tabs for indentation.